libc/socket/unix-socket.c

```diff
@@ -1478,6 +1479,20 @@ void runner(void)
 
 int main(int argc, char *argv[])
 {
+	/* Due to scheduling delays of host system on which emulator runs,
+	 * add some extra value to poll timeout checks
+	 */
+	for (int i = 1; i < argc; i++) {
+		if (strcmp(argv[i], "--extra-poll-delay-ms") == 0) {
+			pollTimeoutDelay = atoi(argv[i + 1]);
+			if (pollTimeoutDelay <= 0) {
+				fprintf(stderr, "--extra-poll-delay-ms argument is not positive integer\n");
+				exit(EXIT_FAILURE);
+			}
+			break;
+		}
+	}
+
 	/* Assume /tmp dir is missing */
 	int isMissing = 0;
```

Comment:
```suggestion
	bool wasTmpMissing = false;
```

Please see the comment above.